ci/docker/install/ubuntu_rat.sh

```diff
@@ -30,22 +33,4 @@ echo "mvn install"
 mvn -Dmaven.test.skip=true install #>/dev/null
 
 echo "build success, cd into target"
-cd apache-rat/target
-
-
-echo "-------Run Apache RAT check on MXNet-------"
-
-#Command has been run twice, once for the logs and once to store in the variable to parse.
-java -jar apache-rat-0.13-SNAPSHOT.jar -E /work/mxnet/tests/nightly/apache_rat_license_check/rat-excludes -d /work/mxnet
-OUTPUT="$(java -jar apache-rat-0.13-SNAPSHOT.jar -E /work/mxnet/tests/nightly/apache_rat_license_check/rat-excludes -d /work/mxnet)"
-SOURCE="^0 Unknown Licenses"
-
-
-echo "-------Process The Output-------"
-
-if [[ "$OUTPUT" =~ $SOURCE ]]; then
-      echo "SUCCESS: There are no files with an Unknown License.";
-else
-      echo "ERROR: RAT Check detected files with unknown licenses. Please fix and run test again!";
-      exit 1
-fi
\ No newline at end of file
+cd apache-rat/target
```

Comment:
newline

Reply:
And why do we cd at the end of an installation shell script?

Reply:
Sorry, was copy&paste. Removed